### DISCUSSION TITLE
Reduce search delay

### DISCUSSION THREAD

**Discussion by praveshishere** [NvChad/NvChad#2693] [Q&A] [answered]
Hi, I am trying to search for files in my project, 
though the results do appear only after 3 to 5 seconds later.

I am not sure if this is some sort of debouncing or configuration issue with my instance.

It happens in a Vue2 project, at others it seems fine but I am not sure.

The project is large, though VSCode does it instantly

**Comment by siduck** [comment, marked as answer]
@praveshishere not a nvchad issue, try using fzf native plugin for telescope

```lua
{
    "nvim-telescope/telescope.nvim",
    opts = { extensions_list = { "fzf"  } },
    dependencies = {
      { "nvim-telescope/telescope-fzf-native.nvim", build = "make" },
    },
  }
  ```

**Reply by praveshishere** [reply]
Thanks @siduck, I am new to vim,
should I add it to chadrc.lua ?

**Reply by siduck** [reply]
@praveshishere  you should know where to add plugins in nvchad, read nvchad.com plugin docs

**Reply by praveshishere** [reply]
I did add it to custom/plugins
![image](https://github.com/NvChad/NvChad/assets/95841565/8e248e7e-ae93-48d9-9b1c-d84698cc088f)
then I ran `NvChadUpdate`.

The search delay is still there, I did read the plugin docs, not sure how to proceed ahead.

**Reply by siduck** [reply]
have you tested it after reopening nvim?

**Reply by praveshishere** [reply]
I did though it did not help. happening in my most of vue projects

**Reply by siduck** [reply]
does this happen with non nvchad configs?

**Reply by praveshishere** [reply]
Thanks @siduck,  It works fine now.

I tried installing it without nvchad, and during configuration,
I found that I had been missing some packages
- https://github.com/BurntSushi/ripgrep?tab=readme-ov-file#installation
- https://github.com/sharkdp/fd?tab=readme-ov-file#installation

After installing these, nvchad also works great.
